ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.test.tsx

```diff
@@ -23,9 +23,19 @@ import {
   mockPluginRegistry,
   MockPlugin,
 } from '@perses-dev/plugin-system';
+import { Props as AutoSizerProps } from 'react-virtualized-auto-sizer';
 import { MOCK_TIME_SERIES_QUERY_RESULT, MOCK_TIME_SERIES_DATA } from '../../test';
 import { TimeSeriesChartPanel, TimeSeriesChartProps } from './TimeSeriesChartPanel';
 
+// Mock AutoSizer because it does not play well with jsdom.
```

Comment:
It might make sense to switch these to e2e tests at some point, but for now, just mocking some basics because we're not trying to test that AutoSizer works here.